starfish/test/intensity_table/test_intensity_table_coords.py

```diff
@@ -68,3 +68,50 @@ def test_tranfering_physical_coords_to_intensity_table():
         z_plane = spot[Axes.ZPLANE.value].data
         physical_z = stack.xarray[Coordinates.Z.value][z_plane]
         assert np.isclose(spot[Coordinates.Z.value], physical_z)
+
+
+def test_tranfering_physical_coords_to_expression_matrix():
+    stack_shape = OrderedDict([(Axes.ROUND, 3), (Axes.CH, 2),
+                               (Axes.ZPLANE, 1), (Axes.Y, 50), (Axes.X, 40)])
+
+    physical_coords = OrderedDict([(PhysicalCoordinateTypes.X_MIN, 1),
```

Comment:
Per our conversation last week, I'd encourage testing this with negative coordinates.